tests/python/pants_test/build_graph/test_build_graph.py

```diff
@@ -317,10 +318,7 @@ def test_raise_on_duplicate_dependencies(self):
     self.add_to_build_file('other/BUILD',
                            'target(name="b")')
 
-    with self.assertRaisesRegexp(
-        BuildGraph.TransitiveLookupError,
-        '^Addresses in dependencies must be unique. \'other:b\' is referenced more than once.'
-        '\s+referenced from //:a$'):
+    with self.assertRaisesRegexp(AddressLookupError, '^Addresses in dependencies must be unique.'):
```

Comment:
Again, having both `//:a` and `other:b` were good...

Reply:
This one was just truncation of the message. Fixed.